packages/app/templates/ui-extensions/projects/checkout_ui/tsconfig.json

```diff
@@ -3,7 +3,8 @@
   // About the `react-jsx` tsconfig option, so IDE don't complain about missing react import
   // Changing options here won't affect the build of your extension
   "compilerOptions": {
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "esModuleInterop": true
```

Comment:
Does this only affect to `checkout_ui` extension?

Reply:
The problem appears once you add a `tsconfig` file to the project. The other extension types does not generate that file after finishing the scaffold process. To be honest, I think that we should either add the same `tsconfig` to every template that support `Typescript` or remove it from every template and let the developers to add their custom configuration

Reply:
Gotcha. Consistency in templates is a discussion that we can have separately so we can go ahead with this fix.